src/CLI/actioner/sonic-cli-ifa.py

```diff
@@ -0,0 +1,161 @@
+import sys
+import time
```

Comment:
@spenugondaa : Separate actioner script file needs to be specified for TAM commands (commands in TAM mode)

Reply:
Done